lib/modules/platform/github/common.ts

```diff
@@ -8,6 +8,7 @@ import type { GhRestPr } from './types';
  * @see https://docs.github.com/en/rest/reference/pulls#list-pull-requests
  */
 export function coerceRestPr(pr: GhRestPr | null | undefined): Pr | null {
+  // istanbul ignore if
```

Comment:
should have a test.

Reply:
I'm about to remove this optionality later